extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSStatisticsUtils.scala

```diff
@@ -24,13 +24,18 @@ import io.trino.tpcds.Table._
 // Page 42 Table 3-2 Database Row Counts
 object TPCDSStatisticsUtils {
 
+  val TINY_SCHEMA_NAME = "tiny"
+
+  val TINY_SCALE_FACTOR = 0.01
+
   val SCALES: Array[Int] = Array(0, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000)
 
   // https://github.com/Teradata/tpcds/issues/26
-  def numRows(table: Table, scale: Int): Long = {
-    require(SCALES.contains(scale), s"Unsupported scale $scale")
+  def numRows(table: Table, scale: Double): Long = {
+    require(SCALES.contains(scale) || scale == TINY_SCALE_FACTOR, s"Unsupported scale $scale")
```

Comment:
it's not a good practice to use `==` to compare two double values, we should always use `abs(d1 - d2) < epsilon` for that.